### PR TITLE
Storage get count

### PR DESCRIPTION
**Pull request by mrdoomus**
=== 1st Pull Request ===
**[Upload]:** Whole API folder with blueprint scripts.

**[Add]:** - New members in README
- Added get and count functions for filestorage and db_storage

**[Fix]:** Pep8 style on various scripts.

### CODE REVIEW

**Comment by mrdoomus** [comment]
Forgot to mention that tests for get/count in test_file_storage and test_db_storage were also added!

File: tests/test_models/test_engine/test_file_storage.py [hunk 1]
```diff
@@ -113,3 +113,38 @@ def test_save(self):
...
+        storage = FileStorage()
+        self.assertEqual(storage.get("State", "This is false"), None)
+
+    def test_count(self):
+        """Test the count of a class"""
+        self.assertEqual(models.storage.count(), len(models.storage.all()))
+
+    def test_count_class(self):
+        """Test that count the number of objects of one class"""
+        for klass in classes:
```
Comment (Dioni1195) [inline tests/test_models/test_engine/test_file_storage.py]: Hi Camilo, I don´t understand the name 'klass' for the variable. I think we can use another meaningful name

File: api/v1/app.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+from models import storage
+from api.v1.views import app_views
+
+app = Flask(__name__)
+app.register_blueprint(app_views)
+
+
+@app.teardown_appcontext
+def teardown(self):
+    storage.close()
```
Comment (Dioni1195) [inline api/v1/app.py]: Hi Camilo, documentation is missing

File: api/v1/views/index.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/python3
+"""  """
+from flask import Flask, jsonify
+from api.v1.views import app_views
+from models import storage
+
+
+@app_views.route('/status')
+def get_status():
+    return jsonify({'status': 'OK'})
```
Comment (Dioni1195) [inline api/v1/views/index.py]: Hi Camilo, Documentation is missing

File: api/v1/views/index.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+
+@app_views.route('/stats')
+def get_stats():
+    return jsonify(amenity=storage.count('Amenity'),
```
Comment (Dioni1195) [inline api/v1/views/index.py]: Hi Camilo, Documentation is missing

File: tests/test_models/test_engine/test_db_storage.py [hunk 5]
```diff
@@ -86,3 +86,37 @@ def test_new(self):
...
+        for klass in classes:
+            self.assertIs(type(models.storage.count(klass)), int)
+
+    def test_correct_count_return(self):
+        """ Test the return value of count """
+        self.assertEqual(models.storage.count(), len(models.storage.all()))
+
+    def test_correct_count_class_return(self):
+        """ Test the return value of count when class is passed """
+        for klass in classes:
```
Comment (Dioni1195) [inline tests/test_models/test_engine/test_db_storage.py]: Camilo this name is not meaningful enough

File: tests/test_models/test_engine/test_file_storage.py (hunk 1 above)
Comment (mrdoomus) [reply tests/test_models/test_engine/test_file_storage.py]: Changed 'klass' for 'class_in_dic'!

File: api/v1/app.py (hunk 2 above)
Comment (mrdoomus) [reply api/v1/app.py]: Added ;)!

File: api/v1/views/index.py (hunk 3 above)
Comment (mrdoomus) [reply api/v1/views/index.py]: Added too ;)!

File: api/v1/views/index.py (hunk 4 above)
Comment (mrdoomus) [reply api/v1/views/index.py]: Added also :D!

File: tests/test_models/test_engine/test_db_storage.py (hunk 5 above)
Comment (mrdoomus) [reply tests/test_models/test_engine/test_db_storage.py]: Changed to 'class_in_dic'

**Review by Dioni1195 — APPROVED** [review]
Ok, I reviewed changes and they look good. Thanks

**Pull request merged**